config/packages/validator.yaml

```diff
@@ -6,7 +7,7 @@ framework:
         # Enables validator auto-mapping support.
         # For instance, basic validation constraints will be inferred from Doctrine's metadata.
         auto_mapping:
-           App\Entity\: []
+           App\Entity\: {}
```

Comment:
This is something the schema will force to change. PHP doesn't care: it is an `array` for it.